chcemvediet/apps/anonymization/cron.py

```diff
@@ -27,7 +28,22 @@ def normalize_pdf(attachment):
     )
     cron_logger.info(u'Normalized attachment: {}'.format(attachment))
 
+def normalize_mocks(attachment, package):
+    with open(u'chcemvediet/apps/anonymization/mocks/normalized.pdf', u'rb') as file:
```

Comment:
Nespoliehal by som sa na to, ze aktualny adresar je root projektu. Ci je to niekde v dokumentacii Django, ze zarucene to bude vzdy aktualny adresar?

Na zistenie absolutnej cesty k rootu projektu radsej pouzi `PROJECT_PATH` zo `settings` a cestu k suboru prilep k nej: `os.path.join(PROJECT_PATH, u'chcemvediet/apps/anonymization/mocks/normalized.pdf'),`

Reply:
Som si este vsimol, ze v subore `chcemvediet/cron.py` robime nieco taketo:
```
logfile = os.path.abspath(os.path.join(os.path.dirname(__file__), u'../logs/mail_admins.log'))
```
Ovela lepsie by bolo pouzit `PROJECT_PATH` aj tu. Pls uprav to aj tu.

Reply:
:ok: